web/src/app/workplace/components/task-master/task-master.component.ts

```diff
@@ -55,19 +55,13 @@ export class TaskMasterComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit() {
-    this.taskService
-      .getSelectedTask()
-      .pipe(takeUntil(this.destroy$))
-      .subscribe((task: Task) => {
-        if (!this.currentBasket) {
-          this.selectedId = task.taskId;
-          this.currentBasket = task.workbasketSummary;
-          this.getTasks();
-        }
-        if (!task) {
-          this.selectedId = '';
-        }
-      });
+    this.taskService.taskChangedStream.pipe(takeUntil(this.destroy$)).subscribe((task: Task) => {
+      this.selectedId = task ? task.taskId : '';
+      if (!this.currentBasket) {
+        this.currentBasket = task.workbasketSummary;
+        this.getTasks();
+      }
+    });
```

Comment:
Why don't we update selectedId and currentBasket like before when taskSelectedStream (getSelectedTask()) changes?

Reply:
I used the taskChangedStream, because i assumed it was the same as the selected stream (it worked for my purposes). I changed it to the selectedStream and the list now gets displayed when working on a task.
Good suggestion, thanks